specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -7535,11 +7541,27 @@
         "modelAsString": true
       }
     },
+    "DiffDiskPlacement": {
+      "type": "string",
+      "description": "Specifies the ephemeral disk placement for operating system disk. This property can be used by user in the request to choose the location i.e, cache disk or resource disk space for Ephemeral os disk provisioning.",
+      "enum": [
+        "CacheDisk",
+        "ResourceDisk"
```

Comment:
We need to explain to the user what is the difference between CacheDisk and ResourceDisk or point them to existing Ephemeral OS Disks documentation at https://docs.microsoft.com/en-us/azure/virtual-machines/windows/ephemeral-os-disks#size-requirements that needs to be updated to explain these options, let us add reference to this doc link and work with PM and Doc team to update this document.

Reply:
updated